src/axom/inlet/examples/example1_expected_documentation.rst

```diff
@@ -0,0 +1,147 @@
+==================
+Input Deck Options
+==================
+.. |uncheck|    unicode:: U+2610 .. UNCHECKED BOX
+.. |check|      unicode:: U+2611 .. CHECKED BOX
+
+--------------
+thermal_solver
+--------------
+
+
+----
+mesh
+----
+
+.. list-table:: Fields
+   :widths: 25 25 25 25 25
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Field Name
+     - Description
+     - Default Value
+     - Range
+     - Required
+   * - filename
+     - file for thermal solver
+     - 
+     - 
+     - |check|
+   * - serial
+     - serial value
+     - 
+     - 
+     - 

```

Comment:
There is something going on with serial and parallel.  Can you look into why they are wrong?

Reply:
Actually parallel looks right but serial is missing the unchecked check box

Reply:
I left required unspecified for serial, and left description for thermal_solver/mesh/parallel unspecified. I can make it say "Unspecified" for clarity, or specify them.

Reply:
unspecified should be the same as false.

Reply:
fixed it